src/PlatformClient.ts

```diff
@@ -34,7 +35,7 @@ export class PlatformClient extends PlatformResources {
         };
 
         if (!this.host) {
-            throw new Error(`CoveoPlatform's host is undefined.`);
+            throw new HostUndefinedError();
```

Comment:
Why this over precedent implementation ?

Reply:
In the process of building my solution, at some point I had another type of error (an `AbortError`). So I thought moving all those errors in the same file would be a cleaner approach. Then I realized that I didn't need the `AbortError`, but I still kept the `HostUndefinedError` in a separate `Errors.ts` file.